src/web/command_control/client/components/RCControllerPanel.tsx

```diff
@@ -60,6 +61,12 @@ export default class RCControllerPanel extends React.Component {
         super(props)
         this.api = props.api
 
+        // Check to see if rc dive parameters are
+        // saved in local storage
```

Comment:
I think they are saved in state and not local storage, but I could be wrong here